docs/guides/platforms/eks-irsa.md

```diff
@@ -0,0 +1,622 @@
+
+Stash expect your database Secret to have `username` and `password` keys. If your database secret does not have them, the `AppBinding` can also help here. You can specify a `secretTransforms` section with the mapping between the current keys and the desired keys.
+
+You don’t need to worry about appbindings if you are using KubeDB. It creates an appbinding containing the necessary informations when you deploy the database. Let’s ensure the appbinding create by `KubeDB` operator.
+
+```bash
+$ kubectl get appbinding -n demo 
+NAME             TYPE                 VERSION   AGE
+sample-mariadb   kubedb.com/mariadb   10.5.8      62m
+```
+
+We have a appbinding named same as database name sample-mariadb. We will use this later for connecting into this database.
+
+### Prepare Backend
+
+We are going to store our backed up data into a [S3 bucket](https://aws.amazon.com/s3/). As we are using irsa, we don't need the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to access the S3 bucket.
+
+At first, we need to create a secret with a Restic password. Then, we have to create a `Repository` crd that will hold the information about our backend storage.
+
+**Create Secret:**
+
+Let's create a secret called `encryption-secret` with the Restic password,
+
+```bash
+$ echo -n 'changeit' > RESTIC_PASSWORD
+$ kubectl create secret generic -n demo encryption-secret \
+    --from-file=./RESTIC_PASSWORD \
+secret "encryption-secret" created
+```
+
+Verify that the secret has been created successfully,
+
+```bash
+$ kubectl get secret -n demo encryption-secret -o yaml
+```
+
+```yaml
+apiVersion: v1
+data:
+  RESTIC_PASSWORD: Y2hhbmdlaXQ=
+kind: Secret
+metadata:
+  creationTimestamp: "2019-07-22T08:49:20Z"
+  name: encryption-secret
+  namespace: demo
+  resourceVersion: "33237"
+  selfLink: /api/v1/namespaces/demo/secrets/encryption-secret
+  uid: 98f12a14-ac5d-11e9-8128-42010a800069
+type: Opaque
+```
+
+**Create Repository:**
+
+Now, let's create a `Repository` with the information of our desired S3 bucket. Below is the YAML of `Repository` crd we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1alpha1
+kind: Repository
+metadata:
+  name: s3-repo
+  namespace: demo
+spec:
+  backend:
+    s3:
+      endpoint: 's3.amazonaws.com'
+      bucket: stash-qa
+      region: us-east-1
+      prefix: /demo/mariadb
+    storageSecretName: encryption-secret
+
+```
+
+Let's create the `Repository` we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/stashed/docs/raw/{{< param "info.version" >}}/docs/examples/guides/platforms/eks/irsa/repository.yaml
+repository.stash.appscode.com/gcs-repo created
+```
+
+Now, we are ready to backup our sample data into this backend.
+
+### Create Policy
+
+We need an IAM policy with proper access permission that will be attached with a IAM service account. Let's create a policy `bucket-accessor` with full access permission to all S3 resources.
+
```

Comment:
A screenshot of the Policy creation UI will be helpful. Also, specify that the policy  has been created from UI.